contracts/src/v0.7/vendor/Owned.sol

```diff
@@ -0,0 +1,63 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.7.0;
```

Comment:
Instead of introducing this file, use `ConfirmedOwner.sol` instead. It's the same thing but with a more explicit name.

Reply:
done